test/e2e/prerender.test.ts

```diff
@@ -957,18 +957,21 @@ describe('Prerender', () => {
       })
     })
 
-    it('should show warning when large amount of page data is returned', async () => {
-      await renderViaHTTP(next.url, '/large-page-data')
-      await check(
-        () => next.cliOutput,
-        /Warning: data for page "\/large-page-data" is 256 kB which exceeds the threshold of 128 kB, this amount of data can reduce performance/
-      )
-      await renderViaHTTP(next.url, '/blocking-fallback/lots-of-data')
-      await check(
-        () => next.cliOutput,
-        /Warning: data for page "\/blocking-fallback\/\[slug\]" \(path "\/blocking-fallback\/lots-of-data"\) is 256 kB which exceeds the threshold of 128 kB, this amount of data can reduce performance/
-      )
-    })
+    if (!isDeploy) {
```

Comment:
drive-by fix since this is reliably failing when I used this file as my test subject